apps/mail/app/(routes)/settings/categories/page.tsx

```
@@ -76,124 +66,119 @@
       }`}
     >
-      <div className="flex items-center justify-between mb-2">
+      <div className="mb-2 flex items-center justify-between">
         <div className="flex items-center gap-2">
           <div
             {...attributes}
             {...listeners}
-            className="cursor-grab active:cursor-grabbing p-1 rounded hover:bg-muted/50 transition-colors"
+            className="hover:bg-muted/50 cursor-grab rounded p-1 transition-colors active:cursor-grabbing"
             aria-label="Drag to reorder"
           >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)